integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/connector/PlatformAccessConfigsTestDto.java

```diff
@@ -72,7 +72,7 @@ public PlatformAccessConfigsTestDto withAvailabilityZone(String availabilityZone
 
     @Override
     public void cleanUp(TestContext context, CloudbreakClient cloudbreakClient) {
-        LOGGER.debug("this entry point does not have any clean up operation");
+        LOGGER.warn(String.format("Cleanup WARN: 'cleanUp' is not implemented for TestDto: %s with name: %s", this, getName()));
```

Comment:
I think in this case we should simply not override the default cleanUp

Reply:
So we should remove all the `cleanUp' is not implemented` `Override` from the project?

Reply:
I think yes, and move this message to the default implementation. This way we will have less code, and truly standardized cleanUp